examples/README.md

```diff
@@ -0,0 +1,30 @@
+# sdl3-sys: Low level Rust bindings for SDL 3
```

Comment:
Not sure it makes sense to copy the README into here
Plus we'd have to keep it up to date
Maybe just point to https://github.com/maia-s/sdl3-sys-rs/tree/main/sdl3-sys from the main README instead? These notes aren't specific to examples